src/Repository/YesplanEventRepository.php

```diff
@@ -62,16 +62,36 @@ public function findNewProductionOnlineEvents(): array
         $conn = $this->getEntityManager()->getConnection();
 
         $sql = '
-        SELECT y.id FROM yesplan_event y LEFT JOIN asana_event a ON y.id=a.id WHERE y.production_online = true AND  (a.created_in_new_events is null OR a.created_in_new_events = 0)
+        SELECT y.id FROM yesplan_event y LEFT JOIN asana_event a ON y.id=a.id WHERE y.production_online = true AND  (a.created_in_new_events is null OR a.created_in_new_events = 0) AND y.profile_id = :profileIdIntern
             ';
         $stmt = $conn->prepare($sql);
-        $stmt->execute([]);
+        $stmt->execute(['profileIdIntern' => '641982209-1522078909']);
+
+        return $stmt->fetchAll();
+    }
+    /**
+     * Finds all events with productionOnline = 1 and profile extern, intern and free events.
+     *
+     * @return YesplanEvent[]
+     */
+    public function findNewProductionOnlineIncludingGratisandExternEvents(): array
+    {
+        //get all events with productiononline = 1 not already created in Asana
+        // returns an array of event id's
+
+        $conn = $this->getEntityManager()->getConnection();
+
+        $sql = '
+        SELECT y.id FROM yesplan_event y LEFT JOIN asana_event a ON y.id=a.id WHERE y.production_online = true AND  (a.created_in_new_events_external is null OR a.created_in_new_events_external = 0) AND (y.profile_id = :profileIdIntern OR y.profile_id = :profileIdEkstern OR y.profile_id = :profileIdGratis)
+            ';
+        $stmt = $conn->prepare($sql);
+        $stmt->execute(['profileIdIntern' => '641982209-1522078909', 'profileIdEkstern' => '65913345-1523601817', 'profileIdGratis' => '94388993-1526294254']);
```

Comment:
These values (`641982209-1522078909`, …) should not be hardcoded here. Use configuration or class `const`s.